docs/introduction.html

```diff
@@ -144,13 +144,13 @@ <h4 class="anchor-heading">
     </li>
   </ul>
   <p>
-    <strong>Producers</strong> are those client applications that publish (write) events to Kafka, and <strong>consumers</strong> are those that subscribe to (read and process) these events. In Kafka, producers and consumers are fully decoupled and agnostic of each other, which is a key design element to achieve the high scalability that Kafka is known for. For example, producers never need to wait for consumers. Kafka provides various <a href="/documentation/#intro_guarantees">guarantees</a> such as the ability to process events exactly-once.
+    <strong>Producers</strong> are those client applications that publish (write) events to Kafka, and <strong>consumers</strong> are those that subscribe to (read and process) these events. In Kafka, producers and consumers are fully decoupled and agnostic of each other, which is a key design element to achieve the high scalability that Kafka is known for. For example, producers never need to wait for consumers. Kafka provides various <a href="/documentation/#impl_guarantees">guarantees</a> such as the ability to process events exactly-once.
```

Comment:
I wonder if pointing to https://kafka.apache.org/documentation/#semantics would actually make more sense. WDYT?